dask_sql/server/app.py

```diff
@@ -74,6 +75,8 @@ async def query(request: Request):
     """
     try:
         sql = (await request.body()).decode().strip()
+        # required for JDBC Driver
+        sql = adjust_for_presto_sql(sql)
```

Comment:
If this is mandatory for compatibility with JDBC driver and is always occurring, perhaps we should just perform the adjustment directly here?